### ISSUE TITLE
Svelte guide

### ISSUE THREAD

**Issue by 1Aron** [master-co/styles#7]
### Discussed in https://github.com/master-co/styles/discussions/4

<div type='discussions-op-text'>

<sup>Originally posted by **guillaume-net** March 24, 2022</sup>
Would it be possible to add a svelte guide?</div>

**Comment by Easy-Cloud-in** [comment]
How to use it svelte and in Astro. These two are gaining more attractions in the community.

**Comment by 1Aron** [comment]
I agree with you very much. We are preparing these two documentation. 
Master Styles can only run on the client side. 
You can directly import it as usual. It has nothing to do with the framework integration. 

We will launch the SSR features next, so stay tuned. 🥳

**Comment by 0Lola** [comment]
Svelte guide has been updated in [Setup in Svelte](https://docs.master.co/styles/setup/svelte). 🎉

If you have any questions, please let me know.
Thanks. 🙏
